### PR TITLE
feat: Creating a function to verify if the wallet is funded and addin…

### PR DESCRIPTION
**Pull request by lgahdl**
…g it to the wallet connection;

### CODE REVIEW

File: src/components/auth/ConnectStellarWallet.tsx [hunk 1]
```diff
@@ -35,6 +37,14 @@ export const ConnectStellarWallet = ({
       onClick={async (e: any) => {
         kit.setWallet(ALBEDO_ID);
         const { address } = await kit.getAddress();
+        const isFunded = await isFundedStellarWallet(address);
+        if (!isFunded) {
+          toast.error(
+            "Can't find your wallet registry, make sure you're trying to connect a funded wallet"
```
Comment (FrancoAguzzi) [inline src/components/auth/ConnectStellarWallet.tsx:43]: I wonder if this is clear for any user / visitor.

Maybe, a text like "Can't find your wallet registry, make sure you're trying to connect a wallet that has XLM funds"

What do you think?

File: src/lib/stellar/isFundedStellarWallet.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    const horizonServer = new StellarSdk.Horizon.Server(
+      "https://horizon.stellar.org"
+    );
+    const account = await horizonServer.loadAccount(address);
+    if (!!account) {
+      return true;
+    }
+    throw new Error("Not able to load account using horizon server");
+  } catch (error) {
+    return false;
```
Comment (FrancoAguzzi) [inline src/lib/stellar/isFundedStellarWallet.ts:14]:
```suggestion
  } catch (error) {
    throw new Error("Failed to load user's Stellar account from Horizon server: ", error)
    return false;
```

File: src/components/auth/ConnectStellarWallet.tsx (hunk 1 above)
Comment (lgahdl) [reply src/components/auth/ConnectStellarWallet.tsx:43]: If you fund a wallet and then transfer all of your XLM funds, you will have 0 XLM, but the wallet will have a registry and will be valid, but maybe there is a better word than "funded" for that, maybe "initialized"?

Comment (lgahdl) [reply src/components/auth/ConnectStellarWallet.tsx]: Replaced the error message for a new one, let me know what you think

**Review by FrancoAguzzi — APPROVED** [review]

**Pull request merged**